16.cpp

```diff
@@ -0,0 +1,41 @@
+// https://leetcode-cn.com/problems/3sum-closest/
+#include <limits.h>
+
+#include <algorithm>
+#include <iostream>
+#include <vector>
+
+class Solution {
+ public:
+  int threeSumClosest(std::vector<int>& nums, int target) {
+    sort(nums.begin(), nums.end());
+    int last_num = 999;
+
+    int len = nums.size();
+    int ans = nums[0] + nums[len - 1] + nums[len - 2];
+    int min_diff = abs(target - ans);
+    for (int i = 0; i < len; ++i) {
+      if (nums[i] == last_num) {
+        continue;
+      }
```

Comment:
???

Reply:
if one number equal with last number, skip it